include/a3/ui_f/hpp/defineResincl.inc

```diff
@@ -1682,7 +1696,7 @@ enum
 #define IDC_MP_QUICK_MAX_PING         111
 #define IDC_MP_QUICK_BEST_SERVER      112
 #define IDC_MP_QUICK_FIND_SERVER      113
-// #define IDC_MP_QUICK_SHOW_SERVERS     114
+#define IDC_MP_QUICK_SHOW_SERVERS     114
```

Comment:
This was commented out due to duplicates in BI defines which HEMTT does warn about.